cmd/aro/operator.go

```diff
@@ -190,7 +190,7 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 			client)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller %s: %v", autosizednodes.ControllerName, err)
 		}
-		if err = (machinehealthcheck.NewReconciler(
+		if err = (machinehealthcheck.NewMachineHealthCheckReconciler(
```

Comment:
This refactor is also going against our other controller patterns of always using the constructor name `NewReconciler`, see above.